common/models/key-value-model.js

```diff
@@ -1,36 +1,116 @@
 var g = require('strong-globalize')();
 
+/**
+ * Data model for key-value databases.
+ *
+ * @class KeyValueModel
+ * @inherits {Model}
+ */
+
 module.exports = function(KeyValueModel) {
-  // TODO add api docs
+  /**
+   * Return the value associated with a given key.
+   *
+   * @param {String} key Key to use when searching the database.
+   * @options {Object} [options] Placeholder for optional settings.
+   * @callback {Function} callback Function to execute upon completion.
+   * @param {Error} err Error object.
+   * @param {*} result Value associated with the given key.
+   * @promise
+   */
   KeyValueModel.get = function(key, options, callback) {
     throwNotAttached(this.modelName, 'get');
   };
 
-  // TODO add api docs
+  /**
+   * Persist a value and associate it with the given key.
+   *
+   * @param {String} key Key to associate with the given value.
+   * @param {*} value Value to persist.
+   * @options {Object} [options]
+   * @property {Number} ttl Time to live for the key-value pair in milliseconds.
+   * @callback {Function} callback The function to execute upon completion.
+   * @param {Error} err Error object.
+   * @promise
+   */
   KeyValueModel.set = function(key, value, options, callback) {
     throwNotAttached(this.modelName, 'set');
   };
 
-  // TODO add api docs
+  /**
+   * Set the TTL (time to live) in ms (milliseconds) for a given key. TTL is the
+   * remaining time before a key-value pair is discarded from the database.
+   *
+   * @param {String} key Key to use when searching the database.
+   * @param {Number} ttl The TTL in ms to set for the key.
+   * @options {Number|Object} [options] Optional settings for the key-value
+   *   pair. If a Number is provided, it is set as the TTL in ms for the
+   *   key-value pair.
+   * @property {Number} ttl TTL in ms for the key-value pair.
```

Comment:
This looks like a mistake to me.

`@options {Number|Object}` is supported by `set`, not by `expire.

IIRC, `expire` does not have any `options` yet and always expect `ttl` in the second argument.